### DISCUSSION TITLE
Session cookies for subdomains?

### DISCUSSION THREAD

**Discussion by xodshi** [blitz-js/blitz#1557] [Q&A] [answered]
I see that during setSessionCookie method domain is not set which means cookie will apply only to the current domain.

Is it possible to set session cookies to ".domain.com" currently?

**Comment by flybayer** [comment]
It's currently not possible, but I think we can expose a config for that. I'm double checking with our auth expert first

**Comment by flybayer** [comment, marked as answer]
Ok yes we can do this! I've opened an issue so that anyone can make a PR for this: https://github.com/blitz-js/blitz/issues/1664
